FlowCrypt/Models/Common/Keypair.swift

```diff
@@ -16,6 +16,7 @@ struct Keypair {
     var source: String
     var allFingerprints: [String]
     var allLongids: [String]
+    var lastModified: Int?
```

Comment:
You've added a value but I didn't see any migration to ensure these values are filled for already existing keys

Reply:
As it's nullable, we don't need new migration.
If this value is nil, update logic happens .

Reply:
I'd rather that you do a migration where you parse all existing keys. See https://github.com/FlowCrypt/flowcrypt-ios/pull/1510#discussion_r860691290 then you don't need an optional, which is better.

Reply:
ok